packages/datadog-plugin-redis/src/index.js

```diff
@@ -73,6 +66,11 @@ class RedisPlugin extends Plugin {
       span.finish()
     })
   }
+
+  configure (config) {
+    Plugin.prototype.configure.call(this, config)
+    return normalizeConfig(this.config)
```

Comment:
Returning here has no effect since the parent method doesn't return. There might be a test missing that should be failing when the config is not normalized properly. Also, it might make more sense to normalize _before_ calling the parent method since the normalization might change properties of the config object that the parent method uses.